.gitignore

```diff
@@ -12,6 +12,8 @@ dist
 dist-ssr
 *.local
 
+chromewebdata_2024-02-23_13-47-38.report.html
```

Comment:
Maybe make this use wildcard characters so it ignores all chromewebdata reports.
i.e.
`chromewebdata_*.report.html`